lib/ASRouterTargeting.jsm

```diff
@@ -67,6 +67,9 @@ const TargetingGetters = {
       update: settings.update
     };
   },
+  get currentTimestamp() {
```

Comment:
Since this is not really a timestamp, maybe we should rename this to `currentDate` instead. We might want an actual `currentTimestamp` for different targeting in the future